test-cases/tests/backup-restore/j08-verify-user-sso-backup-and-restore.md

```diff
@@ -6,6 +6,7 @@ environments:
 estimate: 1h
 tags:
   - destructive
+  - automated
```

Comment:
I think this may have to be removed, as the current process for generating manual testcases excludes any testcases with the `automated` tag